include/TMU.hpp

```diff
@@ -185,7 +191,79 @@ class TMU {
             .Data = CO_LINK(0x2100, 3, 16),
         },
 
-        /**
+        /*
+         * TPDO 1 Settings
+         * 0. The number of sub indices.
+         * 1. The COBID for the transmitting node
+         * 2. The transmission trigger 0xFE is asynchronous
+         * 3. The inhibit time
+         * 4. The event timer
+         */
+        {
+            .Key = CO_KEY(0x1801, 0, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 5,
+        },
+        {
+            //180h+Node-ID
+            .Key = CO_KEY(0x1801, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) CO_COBID_TPDO_DEFAULT(0) + NODE_ID,
+        },
+        {
+            //timer triggered
+            .Key = CO_KEY(0x1801, 2, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0xFE,
+        },
+        {
+            //no inhibit time
+            .Key = CO_KEY(0x1801, 3, CO_UNSIGNED16 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0,
+        },
+        {
+            //send every 2 seconds
+            .Key = CO_KEY(0x1801, 4, CO_UNSIGNED16 | CO_OBJ_D__R_),
```

Comment:
```suggestion
            .Key = CO_KEY(0x1801, 5, CO_UNSIGNED16 | CO_OBJ_D__R_),
```
A value of 4 here is the wrong key for CANOpen, same as above